src/main/java/com/minecolonies/core/colony/buildings/workerbuildings/BuildingBeekeeper.java

```diff
@@ -130,6 +130,17 @@ public void serializeToView(@NotNull final FriendlyByteBuf buf, final boolean fu
         }
     }
 
+    @Override
+    public boolean canEat(final ItemStack stack)
+    {
+        if (stack.getItem() == Items.HONEY_BLOCK)
```

Comment:
Shouldn't this be something like `HONEY_BOTTLE`?
Afaik honey blocks aren't edible

Reply:
idk why it returned to this. I had made it bottle =D Good point

Reply:
#blame-autocomplete 😄